pkg/dt/node.go

```diff
@@ -210,11 +240,35 @@ func (p *Property) AsU64() (uint64, error) {
 	return val, err
 }
 
+type Region struct {
+	Start uint64
+	Size  uint64
+}
+
+// AsRegion converts the property to a Region.
+func (p *Property) AsRegion() (*Region, error) {
```

Comment:
You've done a lot of work here, with nice tests; maybe you could shut this CI up by adding one or two more test cases :-)

Reply:
Done.